openwisp_notifications/models.py

```diff
@@ -26,3 +28,8 @@ class Meta(AbstractIgnoreObjectNotification.Meta):
         swappable = swappable_setting(
             'openwisp_notifications', 'IgnoreObjectNotification'
         )
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    needs_verification = models.BooleanField(default=False)
```

Comment:
Why are you adding this model? There's already a model for EmailAddress from django-allauth, check the openwisp-users module. Are you 100% sure this is necessary? I don't remember asking this in the issue description.